monitor/census.go

```diff
@@ -516,9 +545,16 @@ func InitCensus(nodeType NodeType, version string) {
 			Name:        "transcode_overall_latency_seconds",
 			Measure:     census.mTranscodeOverallLatency,
 			Description: "Transcoding latency, from source segment emerged from segmenter till all transcoded segment apeeared in manifest",
-			TagKeys:     append([]tag.Key{census.kProfiles}, baseTagsWithManifestID...),
+			TagKeys:     append([]tag.Key{census.kProfiles}, baseTags...),
 			Aggregation: view.Distribution(0, .500, .75, 1.000, 1.500, 2.000, 2.500, 3.000, 3.500, 4.000, 4.500, 5.000, 10.000),
 		},
+		{
+			Name:        "transcode_overall_latency_seconds_per_stream",
+			Measure:     census.mTranscodeOverallLatency,
+			Description: "Transcoding latency, from source segment emerged from segmenter till all transcoded segment apeeared in manifest",
+			TagKeys:     append([]tag.Key{census.kProfiles}, baseTagsWithManifestID...),
```

Comment:
I'm not sure if this separation makes sense. A given `manifestId` will always have the same profiles, so this `transcode_overall_latency_seconds_per_stream` will have exactly the same cardinality/values as `transcode_overall_latency_seconds`. 

Or is this `census.kProfiles` not what I think it is?